TData/Compile/Comp10.pas

```diff
@@ -4,12 +4,12 @@
   
 function D2R (deg: real): real;
 begin
-   D2R := deg * PI / 180;
+   D2R := deg * Pi / 180;
 end;
 
 begin
   n := 45;
   WriteLn (n, " degrees = ", D2R (n), " radians.");
-  WriteLn ("Sin (", n, ") = ", sin (D2R (n)));
+  WriteLn ("Sin (", n, ") = ", Sin (D2R (n)));

```

Comment:
The lower case `sin` is deliberate - Pascal is a case-insensitive language and this program should work without changes. If you look back at the lecture video I discussed this very point in my class.

Reply:
Lowercase sin works now